client/app/pages/queries/query.html

```diff
@@ -90,157 +90,56 @@ <h3>
-                      <span ng-if="dataSource.options.doc_url == ''">{{dataSource.type_name}}</span>
-                    </div>
-
-                    <div class="col-xs-7">
-                      <div class="editor__control--right">
-                        <button class="btn btn-default" ng-show="canEdit" ng-click="saveQuery()" title="Save">
-                          <span class="fa fa-floppy-o"></span>
-                          <span class="hidden-xs">Save</span>
-                          <span ng-show="isDirty">&#42;</span>
-                        </button>
-
-                        <button type="button" class="btn btn-primary" ng-disabled="queryExecuting || !canExecuteQuery()" ng-click="executeQuery()">
-                          <span class="zmdi zmdi-play"></span>
-                          <span class="hidden-xs">Execute</span>
-                        </button>
-                      </div>
-                    </div>
-
-                  </div>
-                </div>
-              </div>
-
-            </section>
-          </div>
-
-          <div class="row query-metadata__mobile">
-            <div class="col-xs-4 text-left">
-              <span class="m-r-5">Created by</span>
-              <img ng-src="{{ query.user.profile_image_url }}" class="profile__image_thumb">
-              <strong>
-                <rd-time-ago value="query.created_at"></rd-time-ago>
-              </strong>
-            </div>
-            <div class="col-xs-4 text-center">
-              <span class="m-r-5">Updated by</span>
-              <img ng-src="{{ query.last_modified_by.profile_image_url }}" class="profile__image_thumb">
-              <strong>
-                <rd-time-ago value="query.updated_at"></rd-time-ago>
-              </strong>
-            </div>
-            <div class="col-xs-4 text-right">
-              <span class="query-metadata__property"></span> Refresh Schedule</span>
-              <a ng-click="openScheduleForm()" ng-if="!query.isNew()">{{query.schedule | scheduleHumanize}}</a>
-              <span ng-if="query.isNew()">Never</span>
-            </div>
+          <div class="row editor" resizable r-directions="['bottom']" r-flex="true" resizable-toggle
+               toggle-shortcut="Alt+D" style="min-height: 11px; max-height: 70vh;" ng-if="sourceMode">
+            <query-editor
+              style="width: 100%; height: 100%;"
+              query-text="query.query"
+              autocomplete-query="autocompleteQuery"
+              schema="schema"
+              syntax="dataSource.syntax"
+              is-query-owner="isQueryOwner"
+              update-data-source="updateDataSource"
+              execute-query="executeQuery"
+              can-execute-query="canExecuteQuery"
+              listen-for-resize="listenForResize"
+              save-query="saveQuery"
+              update-query="updateQuery"
+              // LOL angular
```

Comment:
Are comments within tags valid here?

Reply:
I don't remember now. Realizing now I didn't clean up this history -- everything in `query.html` is replaced by `QueryViewTop` and friends now. Guess I shoulda deleted this.